oas_docs/examples/copy_saved_objects_response3.yaml

```diff
@@ -0,0 +1,72 @@
+summary: Failed copy with conflict errors
+description: >
+  A response for a failed copy of a dashboard with the my-dashboard ID including all references from the default space to the marketing and sales spaces. In this example, the dashboard has a reference to a visualization and a Canvas workpad and the visualization has a reference to an index pattern.
+  The result indicates a successful copy for the marketing space and an unsuccessful copy for the sales space because the data view, visualization, and Canvas workpad each resulted in a conflict error.
+  Objects are created when the error is resolved using the resolve copy conflicts API.
```

Comment:
> [!WARNING]  
> Shouldn't we also expand on the error details as in the original docs?
> https://github.com/elastic/kibana/pull/195333/files#diff-025e4cd659a3305e783ecd9d2096f96e229be48ad5014c33254d4179ef1078bfL400-L412?

```
* A {data-source} with the same ID already exists, which resulted in a conflict error. To resolve the error, overwrite the existing object,
or skip the object.

* A visualization with a different ID, but the same origin already exists, which resulted in a conflict error. The `destinationId` field
contains the `id` of the other visualization, which caused the conflict. The behavior is added to make sure that new objects that can be
shared between spaces behave in a similar way as legacy non-shareable objects. When a shareable object is copied into a new space, it
retains its origin so that the conflicts are encountered as expected. To resolve, overwrite the specified destination object, or skip the
object.

* Two *Canvas* workpads with different IDs, but the same origin, already exist, which resulted in a conflict error. The `destinations` array
describes the other workpads which caused the conflict. When a shareable object is copied into a new space, then shared to another space
where an object of the same origin exists, the conflict error occurs. To resolve, pick a destination object to overwrite, or skip the
object.
```

Reply:
IMO the best place for that type of explanation isn't in a reference example like this. Rather, since I assume you can run into the same type of conflicts when doing this task via the UI, I'm thinking it should go into https://www.elastic.co/guide/en/kibana/current/managing-saved-objects.html or a new troubleshooting type of page that can fully explain the cause and solution for each of those errors. If you agree, I'll create an issue so this doesn't get lost when we retire the page.

Reply:
I'm fine with making a separate page

Reply:
Thanks, I've created https://github.com/elastic/docs-content/issues/82 to track that work.